### ISSUE TITLE
Method 'call' in type 'Microsoft.ML.TorchSharp.NasBert.Models.TransformerEncoder' from assembly 'Microsoft.ML.TorchSharp

### ISSUE THREAD

**Issue by aarindam10** [dotnet/machinelearning#6779]
the bug was reported earlier in following post.
https://github.com/dotnet/machinelearning/issues/6665
though the bug was marked as resolved and closed the thread, but still the issues is there. 

Reopening the issue:
----------------------
Now I am using following versions.
 PackageReference Include="Microsoft.ML" Version="2.0.1" 
 PackageReference Include="Microsoft.ML.TorchSharp" Version="0.20.1" 
 PackageReference Include="TorchSharp" Version="0.100.3" 
 PackageReference Include="TorchSharp-cpu" Version="0.100.3" 

still getting following errors

System.TypeLoadException
  HResult=0x80131522
  Message=Method 'call' in type 'Microsoft.ML.TorchSharp.NasBert.Models.TransformerEncoder' from assembly 'Microsoft.ML.TorchSharp, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' does not have an implementation.
  Source=Microsoft.ML.TorchSharp
  StackTrace:
   at Microsoft.ML.TorchSharp.NasBert.NasBertTrainer`2.TrainerBase..ctor(NasBertTrainer`2 parent, IChannel ch, IDataView input)
   at Microsoft.ML.TorchSharp.NasBert.TextClassificationTrainer.Trainer..ctor(NasBertTrainer`2 parent, IChannel ch, IDataView input)
   at Microsoft.ML.TorchSharp.NasBert.TextClassificationTrainer.CreateTrainer(NasBertTrainer`2 parent, IChannel ch, IDataView input)
   at Microsoft.ML.TorchSharp.NasBert.NasBertTrainer`2.Fit(IDataView input)
   at Microsoft.ML.Data.EstimatorChain`1.Fit(IDataView input)
   at Microsoft.ML.Data.EstimatorChain`1.Fit(IDataView input)
   at CustomerReview.CustomerReviewAnalysis.ReviewAnalysis() in F:\MachineL\MLNet\CustomerReviewAnalysis\CustomerReviewAnalysis.cs:line 80
   at Program.<Main>$(String[] args) in F:\MachineL\MLNet\CustomerReviewAnalysis\Program.cs:line 6

**Comment by gabriele-ricci-kyklos** [comment]
@aarindam10 as pointed out in the issue you linked, you need to install the TorchSharp-cpu version 0.98.1: https://github.com/dotnet/machinelearning/issues/6665#issuecomment-1542819386
 I had the exact same issue and solved by installing the correct nuget package version

**Comment by GeorgeS2019** [comment]
@michaelgsharp
Please add deep-learning label to all TorchSharp issues
![image](https://github.com/dotnet/machinelearning/assets/49812372/c4a04e8a-73ef-4046-9948-6db34d528b9d)

**Comment by rosebyte** [comment]
closed as answered
